### DISCUSSION TITLE
Trigger dart code from outside the Python/Flet

### DISCUSSION THREAD

**Discussion by pedmjm** [flet-dev/flet#2434] [Q&A]
### Question

Is it possible to trigger a function/method that's written on flutter/dart outside the running Flet application?

Lets say I have a Flutter and Flet app running on my android device, on Flutter I have this method that uses the speech to text api and returns the value and I process the value using Python. 

How can I trigger the function from within Flet/Python?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 09u2h4n** [comment]
Short answe is no.

Flutter code runs in a Dart virtual machine, and communication between different languages like Python and Dart typically requires some form of inter-process communication or API calls. You might consider using mechanisms like HTTP requests, sockets.
